core/scripts/cli/voting/displayStatus.js

```diff
@@ -0,0 +1,87 @@
+  const filteredRequests = await filterRequests(pendingRequests, account, roundId, roundPhase, voting);
+  const rewards = await getAvailableRewards(web3, voting, account);
+  style.spinnerReadingContracts.stop();
+
+  // If no reveals have taken place in the current vote phase, then
+  // show the global inflation and GAT percentages. Otherwise,
+  // show the round's inflation and GAT percentages.
+  const _inflationRate =
+    roundStats.snapshotId.toString() === "0" ? await voting.inflationRate() : roundStats.inflationRate.toString();
+  const _gatPercentage =
+    roundStats.snapshotId.toString() === "0" ? await voting.gatPercentage() : roundStats.gatPercentage.toString();
+  const inflationRate = parseFloat(web3.utils.fromWei(_inflationRate)) * 100;
+  const gatPercentage = parseFloat(web3.utils.fromWei(_gatPercentage)) * 100;
+
+  // Compute time until next phase and round
+  const { minutesInLastHour, hoursUntilNextPhase, hoursUntilNextRound } = votePhaseTime(currentTime, roundPhase);
+
+  console.group(`${style.success(`\n** Your voting status **`)}`);
+  console.log(`${style.success(`- Current round ID`)}: ${roundId.toString()}`);
+  console.log(
+    `${style.success(`- Current round phase`)}: ${
+      roundPhase.toString() === VotePhasesEnum.COMMIT ? "Commit" : "Reveal"
+    }`
+  );
+  console.log(`${style.success(`- Round Inflation`)}: ${inflationRate.toString()} %`);
+  console.log(`${style.success(`- Round GAT`)}: ${gatPercentage.toString()} %`);
+  console.log(`${style.success(`- Contract time`)}: ${style.formatSecondsToUtc(currentTime)}`);
+  console.log(
+    `${style.success(
+      `- Time until ${roundPhase.toString() === VotePhasesEnum.COMMIT ? "Reveal" : "Commit"}`
+    )} phase: ${hoursUntilNextPhase} hours, ${minutesInLastHour} minutes`
+  );
+  console.log(
+    `${style.success(`- Time until next voting round`)}: ${hoursUntilNextRound} hours, ${minutesInLastHour} minutes`
+  );
+
+  // Display pending requests in a table
+  console.log(
+    `${style.success(`- Pending ${roundPhase.toString() === VotePhasesEnum.COMMIT ? "Price" : "Reveal"} Requests`)}:`
+  );
+  if (filteredRequests.length > 0) {
+    const requestsTable = [];
+    filteredRequests.forEach(request => {
+      const identifierUtf8 = web3.utils.hexToUtf8(request.identifier);
+      const timestampUtc = style.formatSecondsToUtc(parseInt(request.time));
+      requestsTable.push({
+        Identifier: identifierUtf8,
+        Time: timestampUtc
+      });
+    });
+    console.table(requestsTable);
+  }
+
+  // Display rewards to be retrieved in a table
+  console.log(`${style.success(`- Voting Rewards Available`)}:`);
+  if (rewards.roundIds.length > 0) {
+    const rewardsTable = [];
+    Object.keys(rewards.resolvedVotesByRoundId).forEach(roundId => {
+      rewards.resolvedVotesByRoundId[roundId].forEach(_reward => {
+        rewardsTable.push({
+          "Round ID": _reward.roundId,
+          Name: _reward.name,
```

Comment:
nit: why is this capitalized?

Reply:
because all of the labels in shown to user are capitalized, I put this in double quotation marks to make it clearer that it is a table. This is the example output

```
** Your voting status **
  - Current round ID: 9157
  - Current round phase: Reveal
  - Round Inflation: 0.05 %
  - Round GAT: 5 %
  - Contract time: February 23rd 2020, 8:46:35 pm
  - Time until Commit phase: 3 hours, 13 minutes
  - Time until next voting round: 3 hours, 13 minutes
  - Pending Reveal Requests:
  ┌─────────┬──────────────┬───────────────────────────────────┐
  │ (index) │  Identifier  │               Time                │
  ├─────────┼──────────────┼───────────────────────────────────┤
  │    0    │ 'population' │ 'September 30th 2019, 7:20:31 pm' │
  │    1    │ 'population' │ 'September 30th 2019, 7:20:51 pm' │
  └─────────┴──────────────┴───────────────────────────────────┘
  - Voting Rewards Available:
  ┌─────────┬──────────┬────────────────────────────────────────────────┬───────────────┐
  │ (index) │ Round ID │                      Name                      │ Reward Tokens │
  ├─────────┼──────────┼────────────────────────────────────────────────┼───────────────┤
  │    0    │  '9152'  │  'bezos hairs @ August 15th 2019, 7:07:49 am'  │    '50050'    │
  │    1    │  '9156'  │ 'population @ September 23rd 2019, 8:40:51 pm' │    '50050'    │
  └─────────┴──────────┴────────────────────────────────────────────────┴───────────────┘
  
  
```

Reply:
Gotcha. Makes sense.

Reply:
I changed these to lower case, because Prettier complains if I leave the object as:

```
{
          "Round ID": _reward.roundId,
          "Name": _reward.name,
}
```

Prettier wants the "Name" to be without quotation marks, which I think looks weird. So I made all object keys in these tables lowercase